### PR TITLE
Backport Fixes (20.21.7)

### PR DESCRIPTION
**Pull request by devpow112**
Backports #905 and #910.

### CODE REVIEW

File: d2l-rubric-criterion-mobile.js [hunk 1]
```diff
@@ -226,16 +226,16 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-rubric-criterion-mobile">
...
-						criterion-href="[[href]]"
-						assessment-href="[[assessmentCriterionHref]]"
-						token="[[token]]"
-						read-only="[[readOnly]]"
-						score="[[_getVisibleLevelPoints(_selected, _hovered, _focused, _levelEntities, _criterionCells)]]"
-						hovered="[[_hovered]]"
-						assigned-score="[[_score]]"
-					>
-					</d2l-rubric-editable-score>
+					<template is="dom-if" if="[[isNumeric]]">
```
Comment (devpow112) [inline d2l-rubric-criterion-mobile.js:229]: I think this is something that may have come along for the ride but wasn't explicitly in those 2 PR's

**Comment by devpow112** [comment]
Just an FYI, this will need to be manually released since it was a branch made prior to the auto release changes.

File: d2l-rubric-criterion-mobile.js (hunk 1 above)
Comment (MitchelPaulin) [inline d2l-rubric-criterion-mobile.js:229]: I didn't add this template in my PRs, but I think it should still be fine

Comment (devpow112) [reply d2l-rubric-criterion-mobile.js:229]: Yeah looks like it's just the `isNumeric` check stuff

**Review by MitchelPaulin — APPROVED** [review]

**Review by djwester — APPROVED** [review]

**Pull request merged**